src/feedback.js

```diff
@@ -550,46 +551,52 @@ var defaultI18nProperties = require('../i18n/en.properties');
                     const hasNote = $("#feedback-note").val().trim().length > 0;
 
                     const isContactEnabled = $('#feedback-contact-enable').is(':checked');
+                    const isPrivacyPolicy = $('#feedback-privacy-policy').is(':checked');
                     const isValidEmail = utility.isValidEmail($('#feedback-contact-email').val());
                     const hasContactEmail = !isContactEnabled || isValidEmail;
 
-                    if (!hasContactEmail) {
-                        $('#feedback-submit-success,#feedback-submit-error').remove();
-                        $('#feedback-overview-contact-error').show();
-                    } else if (hasTitle && hasNote) {
-                        $('#feedback-submit-success,#feedback-submit-error,#feedback-overview-contact-error').remove();
-                        $('#feedback-overview').hide();
-
-                        post.img = img;
-                        post.title = $('#feedback-title').val();
-                        post.note = $('#feedback-note').val();
-
-                        post.contact = isContactEnabled ? {
-                            name: $('#feedback-contact-name').val() || 'Unknown',
-                            email: $('#feedback-contact-email').val()
-                        } : undefined;
-
-                        var data = {feedback: JSON.stringify(post)};
-                        showSpinner();
-                        (settings.postFunction || $.ajax)({
-                            url: settings.ajaxURL,
-                            dataType: 'json',
-                            type: 'POST',
-                            data: data,
-                            post: post,
-                            success: function () {
-                                hideSpinner();
-                                $('#feedback-module').append(settings.tpl.submitSuccess);
-                            },
-                            error: function () {
-                                hideSpinner();
-                                $('#feedback-module').append(settings.tpl.submitError);
-                            }
-                        });
+                    if (isPrivacyPolicy) {
+                        if (!hasContactEmail) {
+                            $('#feedback-submit-success,#feedback-submit-error').remove();
+                            $('#feedback-overview-contact-error').show();
+                        } else if (hasTitle && hasNote) {

```

Comment:
There is usually a way to write these conditional block so they have a single nesting level (if/else if/..../else). No need to change it.

Reply:
Me too, I'm not sure if the quotes look good. Should I remove them @adrianq?

Reply:
I'm sorry, I haven't thought much about it while writing. I was also thinking of doing it quickly because we are now doing a better feedback component.